.github/docker-images/php5-linux-x64/Dockerfile

```diff
@@ -0,0 +1,74 @@
+FROM amazonlinux:latest
+
+###############################################################################
+# Install prereqs
+###############################################################################
+RUN yum -y update \
+    && yum -y install \
+    tar \
+    git \
+    curl \
+    sudo \
+    # Python
+    python3 \
+    python3-devel \
+    python3-pip \
+    make \
+    cmake3 \
+    gcc \
+    gcc-c++ \
+    openssl-devel \
+    libcurl-devel \
+    libffi-devel \
+    libxml2-devel \
+    sqlite-devel \
+    && yum clean all \
+    && rm -rf /var/cache/yum \
+    && ln -s /usr/bin/cmake3 /usr/bin/cmake \
+    && ln -s /usr/bin/ctest3 /usr/bin/ctest \
+    && cmake --version \
+    && ctest --version
+
+###############################################################################
+# Python/AWS CLI
+###############################################################################
+RUN python3 -m pip install --upgrade pip setuptools virtualenv \
+    && python3 -m pip install --upgrade awscli \
+    && aws --version
+
+###############################################################################
+# Install pre-built OpenSSL
+###############################################################################
+WORKDIR /tmp
+RUN curl -sSL https://d19elf31gohf1l.cloudfront.net/_binaries/libcrypto/libcrypto-1.1.1-linux-x64.tar.gz -o libcrypto.tar.gz \
```

Comment:
Sounds like pretty hardcoded that cloudfront address, hard to change in the future, maybe we can use a CNAME?

Reply:
We don't currently control a domain that we could put the CNAME in.